.travis.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - stage: PythonLint
       os: linux
       python: 3.5
-      install: pip install flake8
+      install: pip install "flake8>=3.7,<3.8"
```

Comment:
It might nice if we could make this a tox job on master, to keep any version-pinning in one place.